packages/extensions-sdk/src/cli/utils/get-template-path.ts

```diff
@@ -1,5 +1,6 @@
-import path from 'path';
+import { dirname, resolve } from 'node:path';
+import { fileURLToPath } from 'node:url';
 
 export default function getTemplatePath(): string {
-	return path.resolve(__dirname, '..', '..', '..', '..', 'templates');
+	return resolve(dirname(fileURLToPath(import.meta.url)), '..', '..', '..', '..', 'templates');
```

Comment:
This resolves the `ReferenceError: __dirname is not defined` error as `__dirname` doesn't readily exist in ESM compared to CJS. This solution references how it's being resolved in the API in:

https://github.com/directus/directus/blob/c301cfbc82d1ce3ba8dad56800552c4899895b6e/api/src/extensions.ts#L68